test/should.js

```diff
@@ -592,6 +650,49 @@ describe('should', function() {
     }, "blah: expected 1 to have property 'length'");
   });
 
+  it('above(n) (dates)', function(){
+    var now = new Date();
+    var oneSecondAgo = new Date(now.getTime() - 1000);
+    var oneSecondAfter = new Date(now.getTime() + 1000);
+
+    (now).should.be.above(oneSecondAgo);
+    (oneSecondAfter).should.be.greaterThan(now);
+    (now).should.not.be.above(oneSecondAfter);
+    (oneSecondAgo).should.not.be.above(oneSecondAfter);
+
+    err(function(){
+      (now).should.be.above(oneSecondAfter, 'blah');
+    }, "blah: expected " +  now.toUTCString() + " to be above " + oneSecondAfter.toUTCString());
+
+    err(function(){
+      (now).should.not.be.above(oneSecondAgo, 'blah');
+    }, "blah: expected " + now.toUTCString() + " to be at most " + oneSecondAgo.toUTCString());
+
+    err(function(){
+      (now).should.have.length.above(3, 'blah');
+    }, "blah: expected " + now.toUTCString() + " to have property 'length'");
+
+    err(function(){
+      ('string').should.have.length.above(now, 'blah');
+    }, "blah: the argument to above must be a number");
+
+    err(function () {
+      (now).should.be.above(1, 'blah');
+    }, "blah: the argument to above must be a date");
+
+    err(function () {
+      (now).should.be.above(null, 'blah');
+    }, "blah: the argument to above must be a date");
+
+    err(function () {
+      (1).should.not.be.above(now, 'blah');
+    }, "blah: the argument to above must be a number");
+
+    err(function () {
+      ([]).should.have.length.above(now, 'blah');
+    }, "blah: the argument to above must be a number");
+  });
+
```

Comment:
Don't need parenthesis around `<var name>` whenever it's `(<var name>).should`; still need parenthesis for the values plugged directly, like `(0).should`